Library/Formula/winetricks.rb

```diff
@@ -1,15 +1,27 @@
 require 'formula'
 
 class Winetricks < ScriptFileFormula
-  homepage 'http://code.google.com/p/winetricks/'
-  url 'http://winetricks.googlecode.com/svn-history/r1217/trunk/src/winetricks', :using => :curl
-  # since the version stated in the field is seldom updated, we append the revision number
-  version '20141125-r1217'
-  sha256 'b2317fbac4282412cc7f6d8f1d387da73dac6d4f5580a8ef8d8f2a2ae8df34b8'
+  homepage 'https://code.google.com/p/winetricks/'
+  url 'https://code.google.com/p/winetricks.git', :revision => 'fa9e42955dbdf780240dedf9057295264fddd98f'
+  version '20150114'
+  sha1 'fa9e42955dbdf780240dedf9057295264fddd98f'
 
-  head 'http://winetricks.googlecode.com/svn/trunk/src/winetricks', :using => :curl
+  head 'https://code.google.com/p/winetricks.git'
 
   depends_on 'cabextract'
+  depends_on 'p7zip'
+  depends_on 'unrar'
+  depends_on 'wine'
+
+  def install
+    bin.install 'src/winetricks'
+    man1.install 'src/winetricks.1'
+    prefix.install 'tests/winetricks-test'
+  end
+
+  test do
+    system "/bin/sh", "#{prefix}/winetricks-test", "quick"
```

Comment:
Seems this may be taking too long: http://bot.brew.sh/job/Homebrew%20Pull%20Requests/20180/version=yosemite/testReport/junit/brew-test-bot/yosemite/test_winetricks/

Anything we can do that's quicker?